packages/@react-spectrum/picker/test/TempUtilTest.test.js

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
reminder: remove this once the select utils v1 is finalized and everything runs clean

Reply:
?

Reply:
these temp util tests were mainly used for testing the select utils in a more focused manner + testing fake/real timers. Was planning on removing them when the api settles and we have more actual usage of the utils but we can always keep the real timers version around for regression testing purposes